tests/ha/ctdb.pm

```diff
@@ -26,11 +26,12 @@ sub run {
 
     # CTDB configuration must be done only in cluster nodes
     if (check_var('CTDB_TEST_ROLE', 'server')) {
-        my $ctdb_cfg = '/etc/samba/smb.conf';
-        my $ctdb_rsc = 'ctdb';
-        my $nmb_rsc  = 'nmb';
-        my $smb_rsc  = 'smb';
-        my $ip_rsc   = 'vip';
+        my $ctdb_cfg    = '/etc/samba/smb.conf';
+        my $ctdb_socket = is_sle('15-SP2+') ? '/var/run/ctdb/ctdbd.socket' : '/var/lib/ctdb/ctdb.socket';
```

Comment:
Shouldn't we decide the `$ctdb_socket` value based on the version of samba? Isn't it possible to have the new value for `ctdb.socket` in older versions of SLES with the newer samba?

Reply:
> Shouldn't we decide the `$ctdb_socket` value based on the version of samba? Isn't it possible to have the new value for `ctdb.socket` in older versions of SLES with the newer samba?

That's the question I asked myself. I don't know if it's possible. I will have the answer once this test will be added in MU process (My next PR). I've checked last samba version used in 15SP1 and it still uses `/var/lib/ctdb/ctdb.socket`.
FMPOV, we can merge it as it is and I will adapt it if we need.